.github/workflows/c-cpp.yml

```diff
@@ -0,0 +1,76 @@
+name: C/C++ CI
+
+on:
+  push:
+    branches: [ '*' ]
```

Comment:
I *think* the `branches` are optional in this case.

Reply:
yep, I simplified the syntax in fb13a87836cc668df9275cd7b17e3baf3d289c07